### ISSUE TITLE
関西Ruby会議08の開催

### ISSUE THREAD

**Issue by ydah** [ruby-no-kai/official#517]
Kyobashi.rb、Ruby関西の高田（@ydah）です。
掲題の通り、関西Ruby会議08の開催を予定し、現在準備を進めております。

* 名称：関西Ruby会議08
* 開催日：2025年6月28(土)
* 会場：[先斗町歌舞練場](https://www.kamogawa-odori.com/kaburenjou/)
* 実行委員長：@ydah
* 主催：Ruby関西、Kyoto.rb、Kobe.rb、Ruby Tuesday、Shinosaka.rb、naniwa.rb、Ruby舞鶴、Kyobashi.rb、AKASHI.rb
* 参加費：無料
* 配信：未定
* サイト： https://regional.rubykaigi.org/kansai08/
* 定員: 400名程度 (会場自体は500名程度入場可能)

「関西Ruby会議」はこれまで7回開催してまいりましたが、大阪以外で開催されたことがなく、「大阪Ruby会議」と名称を変更した経緯があると伺いました。
私は「関西Ruby会議06」に参加したことがきっかけで、Rubyやそのコミュニティに惹かれ、今の自分があると感じています。そのため、この名称を冠した地域Ruby会議を再び開催したいという思いから、「関西Ruby会議」として来年の開催を計画しております。

大阪以外での開催を検討し、関西の他府県の地域Rubyコミュニティとの共催という形で復活させたいと考えており、各所にお声がけをし、調整を進めております。
上記のとおり、まだ詳細が決まっていないため、変更があった場合には改めてご連絡いたします。

つきましては諸々ご協力を仰ぐことになると思いますので、よろしくお願いいたします。

**Comment by kakutani** [comment]
京都!!  例によって会からの支援など相談事項があれば随時よろしくお願いします〜。

**Comment by takahashim** [comment]
会場の予約金の振込依頼が来たので振り込んでおきます

**Comment by hayaokimura** [comment]
@takahashim @snoozer05 
前回同様、メールアドレス（グループ）を利用させていただきたく、設定をお願いしたいです。
`kansai08@rubykaigi.org` で、以下のアドレスをメンバーに入れていただきたいです。
hayao.kimura06 at gmail (hayaokimura)
t.yudai92 at gmail (ydah)
ysk.nano.sub at gmail (k-yoshida)
takashi at bouquetec.com (ogom)
お手数ですが、お願いいたします。

以下去年のやりとりです。
https://github.com/ruby-no-kai/official/issues/497#issuecomment-2090037132

**Comment by takahashim** [comment]
@hayaokimura グループの方対応しますね、少々お待ちいただければ

**Comment by takahashim** [comment]
@hayaokimura `kansai08@rubykaigi.org`のグループ作ってみました。これでいけますかね…？

**Comment by hayaokimura** [comment]
@takahashim 
いけそうです！ありがとうございます！

**Comment by snoozer05** [comment]
@hayaokimura @ydah 準備お疲れ様です！
すみません、1点Rubyの会からのご相談だったのですが、タイムテーブル的にもし今からの差し込みが可能でしたら、どこかで5分程度、Rubyの会でやっていることの紹介やRubyの会からの案内をさせていただく時間を取っていただくことって可能でしょうか...??
こうしたイベントの後援などをやっていることや、るびま・るりまなどの活動、そしてRubyコミュニティ内で何かこまったことがあったときの相談窓口的な役割もやっていますというのを、コミュニティの方々に知っていただく機会を増やしていけるといいなと思っていまして、もし可能そうなら、少しお時間をいただけるとありがたいなと思ったのでした。確認いただけると助かります...! 🙏

**Comment by ydah** [comment]
@snoozer05 ご提案いただきありがとうございます。

日本Rubyの会様の取り組みについてお話しいただくのは、とても意義があることだと思いますので、ぜひ関西Ruby会議としましても、お話いただけるよう調整させていただきたいと思います。

例えばですが、時間帯については朝一番や午後一番ですと人が少なくなる可能性がありますので、クロージング前の時間はいかがでしょうか。問題なければそちらで調整させていただければと思っております。（できるだけ多くの方にお伝えできると良いなと思っております）

**Comment by snoozer05** [comment]
@ydah 早速ありがとうございます！人の多い少ないのご配慮までありがとうございます。時間帯はどこでも大丈夫ですので、そちらで大丈夫でしたらそれでお願いいたします。どうぞよろしくお願いいたします！

**Comment by ydah** [comment]
@snoozer05 クロージングの時間ですと、ご都合が少々気になっておりましたがご快諾いただきありがとうございます。また当日の流れにつきましては確定次第ご連絡致します。どうぞよろしくお願い致します！

**Comment by takahashim** [comment]
メモ: メールにてレントオール江戸川の代金の件で連絡をいただいたので、先ほど振り込んでおきました。

**Comment by pastak** [comment]
1件相談があります。

当会議内で音楽を流す演出を行うつもりで、その関係上利用料などを著作権管理団体に支払うことになる予定があります。「関西Ruby会議08」などの名義で著作権管理団体に利用者登録や手続きをすると、その名前で請求書・領収書なども発行されるようです。
手続きの詳細についてJASRACとNexTONEそれぞれに問い合わせをしているのですが、請求などの名義を分離出来ない場合、「一般社団法人 日本Rubyの会」で領収書などを受け取るために「一般社団法人 日本Rubyの会」で利用者登録などを行う必要がありそうで、このような場合どのように進めていくのが良いでしょうか 🙏

**Comment by takahashim** [comment]
@pastak 遠い昔にJASRACとは取引があったので利用者登録されているかも？（とはいえ住所も変更しているので再登録必要かも） 
NexTONEはなにもないので新規登録から始めることになりそうです。

NexTONEはここから登録しておけばよいでしょうか。
https://www.nex-tone.co.jp/copyright/users/event.html

**Comment by takahashim** [comment]
JASRAC、オンラインアカウントは登録してないですね…（書面でのやりとりでした）。

**Comment by pastak** [comment]
@takahashim ちょうどJASRACからは返事がやってきて、関西Ruby会議08でアカウントを作ると、別途「請求書送付先」を設定できるとのことだったので、一旦その方法で進めようと思います。

**Comment by pastak** [comment]
（コメント消してしまったんですが、JASRACからオンラインアカウントでいけるよと返事が来たんですが、送付先だけじゃなくて領収書の宛名なんですけど・・・って言ったら、ちょっと待ったと来たので、再度確認中です）

**Comment by pastak** [comment]
@takahashim JASRACの京都支部から回答がやってきて、関西Ruby会議08で登録申請して、Rubyの会の方に請求書の送付と宛名の変更の旨を備考欄に書けば、京都支部の方でそのように手続きを進めてもらえることになったので、その形でJASRACの方は進めようと思います。
請求書などの送付先の住所ですが、僕が知っているRubyKaigiのNOC関連の領収書送付先と同じで大丈夫ですか？

**Comment by takahashim** [comment]
はい、同じはずですが https://scrapbox.io/ruby-no-kai/%E9%A0%98%E5%8F%8E%E6%9B%B8%E3%81%AE%E9%80%81%E4%BB%98%E6%96%B9%E6%B3%95 になります！

**Comment by pastak** [comment]
NexTONEの方も返信来まして、同様に進められそうなので、こちらで手続き進めておくので、請求書などの受け取りよろしくおねがいします。タイミングなど分かったらお知らせします。

> ご請求先を「一般社団法人日本Rubyの会」様とする場合は、「関西Ruby会議実行委員会」様で利用者情報の登録を行っていただき、その際に請求担当者情報をご入力いただける欄がございますのでそちらへ「一般社団法人日本Rubyの会」様の情報をご入力の上、利用者登録を進めていただければと存じます。

**Comment by ydah** [comment]
@snoozer05 告知についての当日の流れについてDiscordのDMにてご連絡いたしました。お手隙の際にご確認いただけますと幸いです。

**Comment by ogom** [comment]
@takahashim 開催が終了し、会計の締めと請求書の作成を進めます。
今回は費用が例年より多いのと、また多くの企画を進めた為、複数の請求書が作成される見込みです。

**Comment by takahashim** [comment]
JASRACから封書が来てましたが、「当協会に対する著作物使用料のお支払いは必要ございませんでしたので、その旨ご連絡いたします。」とのことでした。対象外理由：著作権法38条に該当だそうです。

**Comment by pastak** [comment]
> 営利を目的とせず、かつ、聴衆又は観衆から料金（いずれの名義をもつてするかを問わず、著作物の提供又は提示につき受ける対価をいう。以下この条において同じ。）を受けない場合には、公に上演し、演奏し、上映し、又は口述することができる。

に該当ということですね（可能性はあるかなと思いつついましたが、JASRAC側から教えてくれるのですね……

**Comment by pupupopo88** [comment]
📝 今いただいている請求書と立替精算周りは今月中に対応する予定

今いただいているもの以外、他に精算・支払いがあるかが知りたいです！

**Comment by hayaokimura** [comment]
対応ありがとうございます！

精算、支払いこれ以上はありません！

**Comment by snoozer05** [comment]
@hayaokimura 確認ありがとうございました!!
